src/components/navigation-bar/navigation-bar.component.js

```diff
@@ -1,29 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import propTypes from '@styled-system/prop-types';
 import AppWrapper from '../app-wrapper/app-wrapper';
 import StyledNavigationBar from './navigation-bar.style';
 
 const NavigationBar = ({
   navigationType = 'light',
   isLoading = false,
   children,
-  ariaLabel
+  ariaLabel,
+  ...props
 }) => {
+  const spacingProps = {};
+
+  Object.keys(props).forEach((propName) => {
+    if (Object.keys(propTypes.space).includes(propName)) {
+      spacingProps[propName] = props[propName];
+    }
+  });
+
   return (
     <StyledNavigationBar
       role='navigation'
       aria-label={ ariaLabel }
       navigationType={ navigationType }
       data-component='navigation-bar'
+      { ...(Object.keys(spacingProps).length && spacingProps) }
```

Comment:
Because `StyledNavigationBar` is the final component I think we could simply spread `...props` into the component like we have elsewhere. What do you think?

If we choose to keep these props separate, there is no need to check the length of `spacingProps`, we can always spread it because it's always an object.

Reply:
The main reason I was against just spreading `...props` was to avoid inadvertently providing a mechanism for consumers passing in something like `style={{...}}`

Reply:
Ah, we use that so users can add any `aria-*`, `data-*`, `as` props that they might require. If we're going to start conditionally adding props maybe we should make a util that caters for this (or stick with the current convention of passing everything).

Reply:
Yeah that's a fair point, as we've already adopted the spreading of all props into the root I'll just go with that as well to be consistent